spec/rubocop/cop/layout/multiline_method_call_indentation_spec.rb

```diff
@@ -99,6 +99,16 @@
       RUBY
     end
 
+    it 'accepts aligned methods when multiline method chain with a block argument and method chain' do
+      expect_no_offenses(<<~RUBY)
+        a(b)
+          .c(
+            d do
+            end.f
+          )
+      RUBY
+    end
```

Comment:
The following test case also seem necessary:

```ruby
        a&.(b)
          .c(
            d do
            end.f
          )
```

Reply:
Indeed it is. I updated this PR.